frontend/assets/stylesheets/components/_grid.scss

```diff
@@ -21,6 +21,14 @@
     }
 }
 
+.grid--slim {
```

Comment:
Not sure on what to call this. This is a modifier to account for when there is only one row of items so we should drop the bottom margin.